spec/2b_tiles.adoc

```diff
@@ -288,24 +288,39 @@ This means that a tile matrix set can be sparse, i.e. not contain a tile for any
 [caption=""]
 .Requirement 54
 ====
-Each tile matrix set in a GeoPackage SHALL be stored in a different tile pyramid user data table or view^<<K17>>^ with a unique name that SHALL be structured consistently with <<tiles_user_tables_data_table_definition>>, <<example_tiles_table_cols>>, and <<example_tiles_table_sql>>. The `id` column of a tiles table or view SHALL be of type INTEGER and SHALL contain unique values for each row.
+A GeoPackage MAY contain tables containing tile pyramid user data for a single tile matrix set.
+Every such tile pyramid user data table SHALL be structured consistently with <<tiles_user_tables_data_table_definition>> and <<example_tiles_table_sql>>.
+A table SHALL have a primary key column of type INTEGER and that column SHALL act as a `rowid` alias.
+====
+
+[NOTE]
+====
+The integer primary key of a tile pyramid user data table allows tiles to be linked to row level metadata records in the `gpkg_metadata` table by rowid <<B5>> values in the `gpkg_metadata_reference` table as described in clause <<_metadata_reference_table>> below.
+====
+
+[[r153]]
+[caption=""]
+.Requirement 153
+====
+A GeoPackage MAY contain views containing tile pyramid user data for a single tile matrix set.
+Every such tile pyramid user data view SHALL have a first column of type INTEGER and that column SHALL contain unique values for each row.
 ====
 
 [NOTE]
 =====
-For a tiles table, the `id` column SHOULD be a primary key to ensure that each value is unique. Using an integer primary key in a tiles table allows tiles to be linked to row level metadata records in the `gpkg_metadata` table by rowid <<B5>> values in the `gpkg_metadata_reference` table as described in clause <<_metadata_reference_table>> below. Since the concept of primary keys does not exist for views in SQLite, this requirement also provides a way to produce a compliant tiles view with a discoverable key-like column^<<K17>>^.
+Since the concept of primary keys does not exist for views in SQLite, this requirement provides a way to produce a compliant tile pyramid user data view with a discoverable key-like column^<<K17>>^.
 =====
 
 [#example_tiles_table_cols,reftext='{table-caption} {counter:table-num}']
 .Tiles Table or View Definition
```

Comment:
Just "Table" (remove "or View")?

Reply:
I can make that change. Do you think a similar change is appropriate for http://www.geopackage.org/spec121/#example_feature_table_cols and http://www.geopackage.org/spec121/#gpkg_attributes_example?

Reply:
Would be good for consistency. In retrospect, its not that important though - OK with your decision.